src/state-schemas/mirrored-state-schema.js

```diff
@@ -143,5 +143,10 @@ export const mirroredStateProperties = {
     type: 'integer',
     default: undefined,
   },
+  kernelState: {
```

Comment:
can move to editor schema

Reply:
Ah yep, good catch. Halfway through this PR I believe I moved the eval frame kernel setting to just a message pass, so in hindsight this def belongs only in editor.